Frameworks/include/NSRunLoop+Internal.h

```diff
@@ -31,7 +31,7 @@
 - (void)_removeInputSource:(NSInputSource*)source forMode:(NSString*)mode;
 - (void)_addObserver:(NSObject*)observer forMode:(NSString*)mode;
 - (void)_removeObserver:(NSObject*)observer forMode:(NSString*)mode;
-- (StrongId<NSArray*>)_statesForMode:(NSString*)mode;
+- (StrongId<NSArray>)_statesForMode:(NSString*)mode;
```

Comment:
ah, I see. This wasn't initially your code.
I'll downgrade it to a non-showstopper, but it might make the memory management here easier to reckon about.